app/server/lib/DocApi.ts

```diff
@@ -951,7 +952,7 @@ export class DocWorkerApi {
     // (Requires a special permit.)
     this._app.post('/api/docs/:docId/assign', canEdit, throttled(async (req, res) => {
       const docId = getDocId(req);
-      const group = optStringParam(req.query.group);
+      const group = optStringParam(req.query.grou, 'group');
```

Comment:
Looks like an important typo got introduced?

Reply:
> Looks like an important typo got introduced?

Yeah. It was fixed in https://github.com/gristlabs/grist-core/pull/655/commits/37267f40718eec11fa265dd94799abe9e31bf4ae.

Reply:
Ah, sorry, need to be more careful about what I'm reviewing.